app/assets/stylesheets/components.scss

```diff
@@ -243,6 +243,25 @@
   max-width: 100%;
 }
 
+.form-group {
+
+  .field_with_errors {
```

Comment:
@vedant-jain03 can you please explain these errors to me? I'm not able to understand what this bot is trying to say 😅, or is it okay to ignore these errors?

Reply:
The codeclimate always leads us to write a production ready code, according to the above statement it is suggesting to write `field-with-errors` instead of `field_with_errors`

Reply:
But this message can be ignored for this PR as `field_with_errors` is from bootstrap!